update_product_prices_product_category/wizard/product_category_price_update_wizard.py

```diff
@@ -0,0 +1,94 @@
+# -*- coding: utf-8 -*-
+import datetime
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+import base64
+import xlrd
+
+
+class ProductCategoryPriceUpdateWizard(models.TransientModel):
+    _name = "product.category.price.update.wizard"
+    _description = "Product category price update wizard"
+
+    increase_price_by = fields.Selection(
+        string='Increase price by',
+        selection=[
+            ('percentage', _('Percentage')),
+            ('fixed', _('Fixed price'))
+        ],
+        default='percentage',
+        required=True
+    )
+
+    purchase_price = fields.Float(
+        string='Purchase price'
```

Comment:
Add a `help=""` note with info that they can use a fixed price or percentage.
Name could probably be better too like `purchase_price_difference` or something 🤔